README.rst

```diff
@@ -1,10 +1,11 @@
-.. image:: docs/assets/datachain-sys-theme.svg
-   :height: 48
-   :alt: DataChain logo
-
+================
```

Comment:
top line looks dodgy on light mode

![image](https://github.com/user-attachments/assets/a3cb8228-23ca-4ff3-97d4-e3da2fb07b8c)

Reply:
No, the line:

![image](https://github.com/user-attachments/assets/4cec9fe4-3b68-46c1-8bd0-d252494300d1)

Reply:
I think it was fine before, I would just remove the extra top line